src/pages/hire-developers/hiring-hassle.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import StopWatch from "../../../static/stopwatch.svg";
```

Comment:
no need to import from static folder you can directly access image like "/stopwatch.svg" in src

Reply:
done